gamedata/alldefs_post.lua

```diff
@@ -82,6 +82,11 @@ function UnitDef_Post(name, uDef)
 		uDef.icontype = name
 	end
 
+	--global physics behavior changes
+	if uDef.health then
+		uDef.minCollisionSpeed = 2.5 -- define the minimum velocity(speed) required for all units to suffer fall/collision damage.
```

Comment:
Consider providing the value in elmo/s like the speeds in unit defs
```suggestion
		uDef.minCollisionSpeed = 75 / Game.gameSpeed -- define the minimum velocity(speed) required for all units to suffer fall/collision damage.
```